students/elmar_m/lesson02/series.py

```diff
@@ -0,0 +1,140 @@
+'''
+File: series.py
+elmar_m / 22e88@mailbox.org
+Lesson02:  Fibonacci Series Exercise
+------------------------------------
+Python module containing functions to generate Fibonacci and Lucas series 
+of integers and return their n'th value. 
+Instead of Fibonacci or Lucas series start values (0 and 1 resp. 2 and 1)
+arbitrary start values can be given also, resulting in series calculated
+accordingly.
+'''
+
+
+def fib(n):
+    '''
+    returns the n'th value of the Fibonacci series of natural numbers.
+    'n' is required to be a positive integer. 
+    '''
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    elif n > 1:
+        startvalues = [0, 1]
+        counter = 0
+        while counter <= n - 2:
+            a = startvalues[0]
+            b = startvalues[1]
+            c = a + b
+            startvalues[0] = b
+            startvalues[1] = c
+            counter += 1
+        return c
```

Comment:
This is a totally valid solution to the problem and your code is highly readable and well written throughout. But since we know that the fibonacci series is naturally recursive, that is: `fib(n) = fib(n-2) + fib(n-1)`, can you think of a more concise way to rewrite this as a recursive Python function (a function that calls itself)?